core/tests/performance_tests/lda_string_classifier_performance_test.py

```diff
@@ -130,7 +131,8 @@ def predict(self, num):
             raise Exception('No classifier found')
         classifier = (
             classifier_registry.Registry.get_classifier_by_algorithm_id(
-                feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput']))
+                feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput'][
```

Comment:
I've changed it in classifier_services for now. If need be, we can make a separate PR to change it everywhere?